pkg/tsdb/azuremonitor/resourcegraph/azure-resource-graph-datasource.go

```diff
@@ -118,31 +120,15 @@ func (e *AzureResourceGraphDatasource) buildQueries(logger log.Logger, queries [
 	return azureResourceGraphQueries, nil
 }
 
-func (e *AzureResourceGraphDatasource) executeQuery(ctx context.Context, logger log.Logger, query *AzureResourceGraphQuery, dsInfo types.DatasourceInfo, client *http.Client,
-	dsURL string, tracer tracing.Tracer) backend.DataResponse {
-	dataResponse := backend.DataResponse{}
-
+func (e *AzureResourceGraphDatasource) executeQuery(ctx context.Context, query *AzureResourceGraphQuery, dsInfo types.DatasourceInfo, client *http.Client,
+	dsURL string, tracer tracing.Tracer) (*backend.DataResponse, error) {
 	params := url.Values{}
 	params.Add("api-version", ArgAPIVersion)
 
-	dataResponseErrorWithExecuted := func(err error) backend.DataResponse {
```

Comment:
Replacing this loses some information about the query that could be useful, wdyt?

Reply:
Do you know how to see this information? I couldn't figure out where to view this meta data

Reply:
weeeeird ill look again